File: API/Dtos/UploadFile.cs
namespace API.Dtos
{
    public class UploadFile
    {
        public IFormFile File { get; set; }
        public string FileType { get; set; }
    }

    public class FileOutput
    {
        public string FileName { get; set; }
        public string Url { get; set; }
    }
}

File: SPA/src/app/_core/models/upload-file.ts
export interface UploadFile {
  file: File,
  fileType: string;
  password?: string;
}

export interface FileOutput {
  fileName: string;
  url: string;
}

export interface NDWord {
  noiDungCanTim: string;
  noiDungThayThe: string;
}

export interface TextAdd {
  textAdd: string;
}
